pkg/reconcilermanager/controllers/secret_test.go

```diff
@@ -48,7 +47,7 @@ const (
 )
 
 var nsReconcilerKey = types.NamespacedName{
-	Namespace: v1.NSConfigManagementSystem,
+	Namespace: configsync.ControllerNamespace,
```

Comment:
I think all of these changes which switch out this constant can be moved to another PR for easy review

Reply:
Thought about that. Sure. Let me do another PR for this change.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1017